### ISSUE TITLE
Timezone offsets cause due dates to be wrongly changed 

### ISSUE THREAD

**Issue by samuelallan72** [ogarcia/trellowarrior#7]
Task warrior stores dates and times as UTC, and trello gives them with a timezone offset according to your local time (maybe that's the local time of who created the card, don't know). Somewhere during syncing, this timezone information is lost and the card due dates get moved back or forward according to the timezone offset. It must be something like this:

- trellowarrior downloads trello cards (printing them in the console shows that timezone info is present there at least)
- checks if modified by comparing to taskwarrior tasks
- XXX here is where maybe tz info is dropped
- since the dates are different, it syncs one way or another with the new date (may be correct for utc tz, but somewhere the timezone is re-interpreted and thus the date is wrong)

I'm in timezone UTC+10:30, and every time i run trellowarrior to sync, it pushes the due dates of cards/tasks forward by several hours (haven't determined if consistent, or by what exact amount each time)

TL;DR: a bug in timezone handling is causing due dates to be wrongly changed on sync when certain conditions are met.

**Comment by swalladge** [comment]
screenshot from trello illustrating what is happening:

![screenshot](https://static.swalladge.id.au/trellowarrior/1521168810.png)

**Comment by samuelallan72** [comment]
It's also possible that this isn't a problem with trellowarrior, but in the python trello client or tasklib.

**Comment by ogarcia** [comment]
TrelloWarrior don't touch dates. It only pick dates from task or trello what should they be in in UTC format.
